### ISSUE TITLE
New Wazuh packages

### ISSUE THREAD

**Issue by jlruizmlg** [xetus-oss/docker-ossec-server#6]
Hello, we have a new Wazuh packages for the version 2.0

[Changelog](https://github.com/wazuh/wazuh/blob/master/CHANGELOG.md)

This new version has differences in structure, folders name and services name, if i have time the next weekend i will send a pull request with the modifications if you don't do before :)

[Repository](https://github.com/wazuh/wazuh-documentation/blob/new_template/source/installation_guide/installing-manager/packages-installation/wazuh_server_deb.rst
)

Thanks for your contribution withe Ossec community !!

**Comment by tkent** [comment]
@jlruizmlg 

I poked around the site a bit - very cool stuff! However, reading [the installation guide](http://documentation.wazuh.com/en/latest/installation.html) and the GitHub repo docs, it looks like these packages are an OSSEC fork. 

Knowing that, I'm inclined to have a separate image for that fork. Especially since the name of the image is is clearly ossec-server, not wazuh-manager :-). 

If I have that right, I'm happy to make a fork of this project for wazuh-manager. I think this image should have the latest ossec-hids releases from wazuh (or a manual build of ossec, if wazuh doesn't keep up the pre-built packages).

If we do go down the fork route, I'll see about dropping in a forwarder we have that makes the ossec logs AlienVault compatible. We had to write it for our infrastructure, and I'd imagine it would come in handy for your users. It just needs some prep before making it public.
